src/services/twitterRSSService.ts

```
@@ -0,0 +1,484 @@
+    if (!this.isLoggedIn) {
+      logger.warn(
+        'Twitter not authenticated - cannot fetch tweets. Check authentication status.'
+      );
+      throw new Error(
+        'Twitter authentication required. Please check credentials and try again.'
+      );
+    }
+
+    const allTweets: TweetData[] = [];
+    const maxTweetsPerList = parseInt(
+      (this.getConfig('MAX_TWEETS_PER_LIST', '50') as string)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)